### DISCUSSION TITLE
新建文章gitlab显示404

### DISCUSSION THREAD

**Discussion by somnifex** [Qexo/Qexo#284] [Q&A]
![](https://s2.loli.net/2023/01/07/N5EKHOAhimYsqZF.png)

**Comment by am-abudu** [comment]
是否可以截图你的gitlab配置？注意将密钥打码处理

**Reply by somnifex** [reply]
已经上传

**Comment by somnifex** [comment]
![IMG_20230107_150820](https://user-images.githubusercontent.com/98788152/211138698-9900f8dd-2498-43b7-b2ee-3d80978541e4.jpg)
这是我的配置

**Comment by am-abudu** [comment]
可以简单描述一下你的操作流程吗，因为我尝试新建文章并没有出现报错。如果是本地部署的话，是否可以贴出控制台日志？

**Reply by somnifex** [reply]
我可能大致猜到是什么原因了，可能跟部署方式有关，我采用的是双备份，修改使用gitlab，但是部署采用的是vercel➕github，这样可能因为同步原因无法发布

**Reply by somnifex** [reply]
也就是gitlab push到GitHub中

**Reply by am-abudu** [reply]
但是发布实际上是编辑的你Gitlab上的文件。这不应该报错的

**Reply by somnifex** [reply]
我今晚回去再把流程走一遍尝试一下

**Reply by somnifex** [reply]
流程：新建文章-随便写点内容-发布-报错404
![](https://s2.loli.net/2023/01/07/sK3loa1zbcCwVjh.png)

**Reply by somnifex** [reply]
gitlab中包含读写权限：
 ![](https://s2.loli.net/2023/01/07/ve9b3WN4DlaUQK7.png)

**Reply by am-abudu** [reply]
我测试的时候是真的没有出错。。。
或者你尝试更新一下Dev？
![](https://user-images.githubusercontent.com/51912589/211143447-bd204139-6aa9-408d-ba5a-bd7e896544f0.png)

**Reply by somnifex** [reply]
我现在猜测的可能是因为gitlab和github同步的原因吗，实在不行我重新部署下看看gitlab直接部署看看

**Reply by am-abudu** [reply]
因为程序并不会直接接触到Github方面，所以我感觉问题不大。或者你可以试试全套都在Github上面能不能正常使用

**Reply by somnifex** [reply]
我尝试一下全部运行在github中，如果全部运行在gitlab我尝试失败了

**Reply by somnifex** [reply]
重新部署解决了，感谢！
